<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthDialog</class>
 <widget class="QDialog" name="AuthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2E3440;
color: #D8DEE9;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #81A1C1;
    font-size: 16pt;
    font-weight: bold;
    qproperty-alignment: AlignCenter;
}</string>
     </property>
     <property name="text">
      <string>Добро пожаловать!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid #4C566A;
    border-radius: 5px;
    margin-top: 10px;
    padding-top: 15px;
}
QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px;
}</string>
     </property>
     <property name="title">
      <string>Детали входа:</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelUsername">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #D8DEE9;
}</string>
        </property>
        <property name="text">
         <string>Логин:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditUsername">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: #3B4252;
    color: #E5E9F0;
    border: 1px solid #4C566A;
    border-radius: 4px;
    padding: 6px;
}</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="styleSheet">
         <string notr="true">QLabel {
    color: #D8DEE9;
}</string>
        </property>
        <property name="text">
         <string>Пароль:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    background-color: #3B4252;
    color: #E5E9F0;
    border: 1px solid #4C566A;
    border-radius: 4px;
    padding: 6px;
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditStatus">
     <property name="styleSheet">
      <string notr="true">QTextEdit {
    background-color: #3B4252;
    color: #A3BE8C;
    border: 1px solid #4C566A;
    border-radius: 4px;
    padding: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonRegister">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #4C566A;
    color: #D8DEE9;
    border-radius: 4px;
    padding: 8px;
    min-width: 100px;
}
QPushButton:hover {
    background-color: #5E81AC;
}</string>
       </property>
       <property name="text">
        <string>Зарегистрироваться</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLogin">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #5E81AC;
    color: white;
    border-radius: 4px;
    padding: 8px;
    min-width: 100px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #81A1C1;
}</string>
       </property>
       <property name="text">
        <string>Войти в аккаунт</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
